include/range/v3/utility/concepts.hpp

```diff
@@ -655,8 +626,13 @@ namespace ranges
         template<typename T, typename...Us>
         using Constructible = concepts::models<concepts::Constructible, T, Us...>;
 
-        template<typename T>
-        using DefaultConstructible = concepts::models<concepts::DefaultConstructible, T>;
+        template<typename T, typename...Us>
+        using ConstructibleRef =
+            meta::apply<
+                meta::if_<std::is_reference<T>,
+                    meta::quote<std::is_constructible>,
+                    meta::quote<Constructible>>,
+                T, Us...>;
```

Comment:
I was thinking more in terms of `Constructible` being implemented in terms of `Initializable`, but adding `Destructible` and dynamic allocation.